plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/CredentialHelperTest.java

```diff
@@ -16,27 +16,40 @@
 
 package com.google.cloud.tools.eclipse.login;
 
-import org.junit.Assert;
-import org.junit.Test;
-
 import com.google.api.client.auth.oauth2.Credential;
 import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
 import com.google.api.client.http.javanet.NetHttpTransport;
 import com.google.api.client.json.jackson.JacksonFactory;
 import com.google.gson.Gson;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.file.Path;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 
 public class CredentialHelperTest {
 
+  @Rule public TemporaryFolder tempFolder = new TemporaryFolder();
+
   @Test
-  public void testGetJsonCredential() {
+  public void testToJsonFile() throws IOException {
     Credential credential = createCredential("fake_access_token", "fake_refresh_token");
-    String jsonCredential = new CredentialHelper().toJson(credential);
+    Path jsonFile = tempFolder.getRoot().toPath().resolve("credential-for-gcloud.json");
```

Comment:
But the default options of Files.write() will ensure that if the file exists, it will be truncated before writing. So in fact it doesn't matter whether it exists or not, it will be either overwritten or created.